### PR TITLE
Run npm audit fix, fix bin linking

### PR DESCRIPTION
**Pull request by Pholey**
Fixes https://github.com/dogescript/dogescript/issues/245

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "optimist": "^0.6.1",
     "xhr": "^2.5.0"
   },
-  "bin": "./bin/dogescript.js",
+  "bin": "./bin/dogescript.bin.js",
```
Comment (AnEmortalKid) [inline package.json]: any reason to not use the `dist/` based file? I believe webpack adds a shebang for a node environment. ?

Comment (vpzomtrrfrt) [reply package.json]: I think doing that would mean we need a prepare script that runs the build

Comment (AnEmortalKid) [reply package.json]: I guess then based on the definition for `bin` from here: https://docs.npmjs.com/files/package.json

We probably should just copy the shebang line and not use webpack: https://github.com/dogescript/dogescript/blob/master/webpack.config.js#L36

Or we should use a prepublish/prepare script then to run webpack and get that shebang added.

Comment (AnEmortalKid) [reply package.json]: @Pholey should we add the shebang to this file then?

**Pull request merged**